diff --git a/CMakeLists.txt b/CMakeLists.txt
index b5de412..034ac12 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,6 @@ else()
     add_compile_options(-Wall -Wextra -pedantic)
 endif()
 
-find_package(NumPy REQUIRED)
 find_package(PythonExtensions REQUIRED)
 find_package(Python COMPONENTS Interpreter Development)
 include(FetchContent)
diff --git a/setup.py b/setup.py
index 0bf720e..b8a4b29 100644
--- a/setup.py
+++ b/setup.py
@@ -1,9 +1,13 @@
 from skbuild import setup
 import rapidfuzz_capi
+import numpy as np
 
 with open('README.md', 'rt', encoding="utf8") as f:
     readme = f.read()
 
+print(f'RF_CAPI_PATH:STRING={rapidfuzz_capi.get_include()}')
+print(f'NumPy_INCLUDE_DIR:STRING={np.get_include()}')
+
 setup(
     name="rapidfuzz",
     version="2.0.11",
@@ -33,6 +37,8 @@ setup(
         "rapidfuzz/distance": ["*.pyi"]
     },
     python_requires=">=3.6",
-
-    cmake_args=[f'-DRF_CAPI_PATH:STRING={rapidfuzz_capi.get_include()}']
+    cmake_args=[
+        f'-DRF_CAPI_PATH:STRING={rapidfuzz_capi.get_include()}',
+        f'-DNumPy_INCLUDE_DIR:STRING={np.get_include()}'
+    ]
 )
